x-pack/plugin/esql/build.gradle

```diff
@@ -121,28 +124,41 @@ tasks.named("test").configure {
           return example;
         })
       }
-      if (count == 0) {
-        logger.quiet("ESQL Docs: No function signatures created. Skipping sync.")
-      } else if (count == 1) {
-        logger.quiet("ESQL Docs: Only files related to $types, patching them into place")
+      if (countSnippets == 0) {
+        logger.quiet("ESQL Docs: No function/operator snippets created. Skipping sync.")
+      } else {
+        logger.quiet("ESQL Docs: Found $countSnippets generated markdown files to patch into docs")
         injected.fs.sync {
-          from functionsFolder
-          into functionsDocFolder
+          from snippetsFolder
+          into snippetsDocFolder
           include '**/*.asciidoc', '**/*.svg', '**/*.md', '**/*.json'
           preserve {
             include '/*.asciidoc', '**/*.asciidoc', '**/*.md', '**/*.json', '**/*.svg', 'README.md'
           }
           filter readExample
         }
+      }
+
+      List images = fileTree(imagesFolder).matching {
+        include "**/*.svg"  // Recursively include all SVG files
+      }.files.collect { it.name }
+      int countImages = images.size()
+      Closure replaceFont = line -> {
+        // The es-docs team has a recommended set of fonts for use with code, and they size similarly to the previous Roboto Mono, which is no longer available in the docs webpage
+        line.replaceAll(/font-family:\s*Roboto Mono[^;]*;/, 'font-family: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;')
```

Comment:
Can this be fixed in its source instead of having a Gradle script updating it?

Reply:
Unfortunately the SVG library that we used seemed to require actually loading fonts from disk (requiring all these fonts to be installed on all developer machines and CI), and also seemed to only support setting one font (although two always appeared in the list). While I could try fight with this in the java code and find workarounds, I felt that since we already had a gradle task for editing files, it seemed easier to just do this edit here (just a one-liner, compared to the probable mess in Java, if even possible). The other change to the SVG (removal of `#guide`) was done in the java code.